tests/puppeteer/campaigns/data/demo/orderStatuses.js

```diff
@@ -1,10 +1,10 @@
 module.exports = {
   Statuses: {
-    awaitingCheckPayment: {index: 1, status: 'Awaiting check payment'},
-    canceled: {index: 6, status: 'Canceled'},
-    paymentAccepted: {index: 7, status: 'Payment accepted'},
-    paymentError: {index: 8, status: 'Payment error'},
-    awaitingBankWire: {index: 10, status: 'Awaiting bank wire payment'},
-    shipped: {index: 14, status: 'Shipped'},
+    awaitingCheckPayment: {id: 1, status: 'Awaiting check payment'},
+    paymentAccepted: {id: 2, status: 'Payment accepted'},
+    shipped: {id: 4, status: 'Shipped'},
+    canceled: {id: 6, status: 'Canceled'},
+    paymentError: {id: 8, status: 'Payment error'},
+    awaitingBankWire: {id: 10, status: 'Awaiting bank wire payment'},
```

Comment:
What happened here ?

Reply:
Index was not used at all and it was referred to the position of status in old orders page. 
So indexes is now ids, and used in function setOrderStatus.